aspnetcore/includes/connectionid-signalr.md

```diff
@@ -0,0 +1,2 @@
+> [!WARNING]
+> Security warning: Exposing `ConnectionID` can lead to malicious impersonation if the SignalR server or client version is ASP.NET Core 2.2 or earlier.
```

Comment:
Nit: I would replace all uses of "ConnectionID" with "ConnectionId" since that's how it's [exposed in the API](https://github.com/dotnet/aspnetcore/blob/c848c33cfa392f118b31523ac7692318ef142d85/src/SignalR/server/Core/src/HubCallerContext.cs#L16-L19).

Generally I use either "ConnectionId" or "connection ID" to refer to this, but I tend to prefer the former.

```suggestion
> Security warning: Exposing `ConnectionId` can lead to malicious impersonation if the SignalR server or client version is ASP.NET Core 2.2 or earlier.
```